src/Refactoring-UI/StMethodNameEditorPresenter.class.st

```diff
@@ -228,6 +228,12 @@ StMethodNameEditorPresenter >> canRenameArgs: anObject [
 	argumentsList items do: [ :arg | arg canBeRenamed: anObject ]
 ]
 
+{ #category : 'accessing' }
+StMethodNameEditorPresenter >> cancelled [
+	
+	^ self withWindowDo: #cancelled
+]
+
```

Comment:
@Ducasse This method is categorized as 'TO REMOVE' in `SpPresenter`, and in there it is missing the `^`, so it doesn't work. Other drivers also send `cancelled` to their presenters. This code should be refactored in a new PR